plopFile.js

```diff
@@ -0,0 +1,46 @@
+const { mkdir } = require("fs/promises");
+const { join } = require("path");
+const { camelCase } = require("case-anything");
+const { createExerciseDirectoryName } = require("./generators/helpers");
+const { writeReadme } = require("./generators/writeReadme");
+const { writeExercise } = require("./generators/writeExercise");
+const { writeExerciseSpec } = require("./generators/writeExerciseSpec");
```

Comment:
Super nit: I would maybe throw all of the generator functions into a barrel file to tidy up the imports here.